packages/matter.js/src/log/Logger.ts

```diff
@@ -86,9 +86,9 @@ function plainLogFormatter(now: Date, level: Level, facility: string, values: an
 }
 
 const ANSI_CODES = {
-    PREFIX: "\x1b[37m\x1b[2m",
-    FACILITY: "\x1b[37m\x1b[1m",
-    LEVEL_DEBUG: "\x1b[37m",
+    PREFIX: "\x1b[90m\x1b[2m",
+    FACILITY: "\x1b[90m\x1b[1m",
+    LEVEL_DEBUG: "\x1b[90m",
```

Comment:
May be good to add comment of what expected color is.

Reply:
Good idea, added